packages/plugin-core/src/_extension.ts

```diff
@@ -451,8 +451,20 @@ export async function _activate(
 
   const { keybindingConfigPath, migratedKeybindings } = KeybindingUtils.checkAndMigrateLookupKeybindingIfExists();
```

Comment:
currently we run this at startup everytime. can we put this logic into the migration so it only runs during upgrade?

Reply:
keybinding migration / conflict resolution is a pretty plugin-y thing to do, do we want the engine-server to run that? We'll have to import vscode and some plugin-core utils to engine-server if we want this as a migration.

I think it would be fine to keep it here, but add another check for `InstallStatus.UPGRADED`. So it only checks during upgrade.